### PR TITLE
Mg/document card

### PR DESCRIPTION
**Pull request by mantagen**

### CODE REVIEW

File: reg-simple-ui/components/DocumentCard/DocumentCard.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { getCountry } from '../../fixtures/countries.js';
```
Comment (Jbarget) [inline reg-simple-ui/components/DocumentCard/DocumentCard.js]: getCountryName?

File: reg-simple-ui/components/DocumentCard/DocumentCard.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { colors } from '../../theme.js';
```
Comment (Jbarget) [inline reg-simple-ui/components/DocumentCard/DocumentCard.js]: use `getColor` from `themeHelpers.js`

Comment (mantagen) [reply reg-simple-ui/components/DocumentCard/DocumentCard.js]: It was before getColor was a thing but will change now

**Comment by mantagen** [comment]
image


FYVP @Jbarget here it is with font + flag

**Pull request merged**